term/src/terminalstate/mod.rs

```diff
@@ -379,8 +379,24 @@ struct UnicodeVersionStackEntry {
 
 fn default_color_map() -> HashMap<u16, RgbColor> {
     let mut color_map = HashMap::new();
+    // Match colors to the VT340 color table:
+    // https://github.com/hackerb9/vt340test/blob/main/colormap/showcolortable.png
     color_map.insert(0, RgbColor::new_8bpc(0, 0, 0));
-    color_map.insert(3, RgbColor::new_8bpc(0, 255, 0));
+    color_map.insert(1, RgbColor::new_8bpc(0x33, 0x33, 0xcc));
```

Comment:
if you fancy a game of Rust golf, this is at a point where I think we could reduce some boilerplate, something like:

```rust
   for (idx, r, g, b) in &[
               (0, 0x00, 0x00, 0x00),
               (1, 0x33, 0x33, 0xcc), 
               (2, 0xcc, 0x23, 0x23), 
     ] {
       color_map.insert(idx, RgbColor::new_8bpc(r, g, b));
   }
```

Reply:
Fore! :)

Reply:
The compiler did not like the "&".  It wanted more like:

```Rust
for (idx, r, g, b) in [
               (0, 0x00, 0x00, 0x00),
               (1, 0x33, 0x33, 0xcc), 
               (2, 0xcc, 0x23, 0x23), 
     ] {
       color_map.insert(idx, RgbColor::new_8bpc(r, g, b));
   }
```

Does that work for you?

Reply:
Yeah, that's totally fine